### PR TITLE
Bump npm packages to remove some deprecated one

### LINKED ISSUE #3733: Npm package `electron-rebuild` is deprecated use `@electron/rebuild`

### LINKED ISSUE #3736: Npm package `asar` is deprecated use `@electron/asar`

### PR DESCRIPTION
**Pull request by ADcorpo**
# What has changed ?
Override `@electron/universal`'s version to stop using deprecated dependency `asar`
Switch from electron-rebuild to @electron/rebuild (package name change, not breaking).

- [ ] Does this PR affect the User ?

fixes #3736 
fixes #3733

### CODE REVIEW

**Review by FirelightFlagboy — APPROVED** [review]

**Comment by FirelightFlagboy** [comment]
I've trigger a manual job for `packaging-ci` https://github.com/Scille/parsec-cloud/actions/runs/3704158235 to see if everything is ok

File: oxidation/client/electron/package.json [hunk 1]
```diff
@@ -35,11 +35,14 @@
...
     },
     "devDependencies": {
+        "@electron/rebuild": "^3.2.10",
         "electron": "^22.0.0",
         "electron-builder": "~23.6.0",
-        "electron-rebuild": "^3.2.9",
         "typescript": "~4.9.3"
     },
+    "overrides": {
+        "@electron/universal": "~1.3.4"
```
[the remark this answers is not in the document]
Comment (ADcorpo) [reply oxidation/client/electron/package.json:44]: They already have the change in https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/package.json I think it's only a matter of releasing it

[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply oxidation/client/electron/package.json:44]: Nice :)

I've created an issue for that

**Comment by FirelightFlagboy** [comment]
> I've trigger a manual job for packaging-ci ...

The job `package electron` finished with a success, so it should be safe to merge :100:

**Pull request merged**